apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html

```diff
@@ -113,4 +121,67 @@
             <button (click)="logout()"><i class="icon-logout mr-2 font-18"></i> {{localeData?.logout}}</button>
         </div>
     </div>
+    <!-- branch design option 2 -->
+    <div class="company-branch-opt-2" *ngIf="activeDesign === 2">
+        <div class="company-list">
+            <ul class="list-style-none font-14"
+                *ngIf="companyListForFilter && companyListForFilter.length && !(isCompanyRefreshInProcess$ | async)">
+
+                <ng-container *ngFor="let company of companyListForFilter; let i = index">
```

Comment:
Avoid using single character variable names as they make searching difficult
```suggestion
                <ng-container *ngFor="let company of companyListForFilter; let companyIndex = index">
```